components/results_table/commons/results_table_base.lua

```diff
@@ -328,6 +330,14 @@ function BaseResultsTable:build()
 		end
 	end
 
+	if self.config.onlyAchievements then
+		displayTable:tag('tr')
+			:tag('th')
+				:attr('colspan', 42)
+				:css('font-style', 'italic')
+				:wikitext('[[' .. self.config.opponent .. '/' .. self.config.resultsSubPage .. '|Extended list of results]]')
```

Comment:
```suggestion
				:wikitext('[[/' .. self.config.resultsSubPage .. '|Extended list of results]]')
```
Wouldn't this work?

Reply:
only if used on the player/team page
if used elsewhere it would just link to the according subpage of the page the achievements display is called